src/cosalib/aws.py

```diff
@@ -123,6 +123,7 @@ def aws_run_ore(build, args):
         '--ami-description', f'{build.summary} {build.build_id}',
         '--file', f"{build.image_path}",
         '--disk-size-inspect',
+        '--arch', f"{args.arch}",
```

Comment:
Shouldn't this switch be added only `if args.arch is not None`?

Reply:
fixed